susemanager-utils/testing/automation/obs-project.py

```diff
@@ -66,7 +66,9 @@ def add(args):
     root.find("title").text = new_title 
 
     if (maintainer!=""):
-        print("DEBUG: Adding user {} as maintainer".format(auth_user))
+        print("DEBUG: Adding user {} as the only maintainer".format(auth_user))
+        for user in root.findall("person"):
```

Comment:
Does this work for both users and groups? If, the PR LGTM :-)

Reply:
good point. I just removed the groups and squashed the new commit.